blocks.md

```diff
@@ -384,40 +396,47 @@ Creates a block which displays disk space information.
 [[block]]
 block = "disk_space"
 path = "/"
-alias = "/"
 info_type = "used"
 unit = "GiB"
-format = "{icon}{used}/{total} {unit} ({available}{unit} free)"
+format = "{icon} {used}/{total} ({available} free)"
 ```
 
 #### Options
 
 Key | Values | Required | Default
 ----|--------|----------|--------
 `alert` | Available disk space critical level as a percentage or Unit. | No | `10.0`
-`alias` | Alias that is displayed for path. | No | `"/"`
-`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{alias} {available} {unit}"`
+`warning` | Available disk space warning level as a percentage or Unit. | No | `20.0`
+`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{alias} {available}"`
 `info_type` | Currently supported options are `"available"`, `"free"`, and `"used"` (sets value for alert and percentage calculation). | No | `"available"`
 `interval` | Update interval, in seconds. | No | `20`
 `path` | Path to collect information from. | No | `"/"`
-`unit` | Unit that is used to display disk space. Options are `"MB"`, `"MiB"`, `"GB"`, `"GiB"`, `"TB"`, `"TiB"` and `"Percent"`. | No | `"GB"`
-`warning` | Available disk space warning level as a percentage or Unit. | No | `20.0`
+`unit` | Unit that is used when `alert_absolute` is set for `warning` and `alert`. Options are `"B"`, `"KB"` `"MB"`, `"GB"`, `"TB"`. | No | `"GB"`
 `alert_absolute` | Use Unit values for warning and alert instead of percentages. | No | `false`
 
+#### Deprecated Options
+Key | Values | Required | Default
+----|--------|----------|--------
+`alias` | Sets the value for `{alias}` placeholder | No | `"/"`
+
+
 #### Available Format Keys
 
-Key | Value
-----|-------
-`{alias}` | Alias for disk path
-`{available}` | Available disk space (free disk space minus reserved system space)
-`{bar}` | Display bar representing percentage
```

Comment:
Is it possible to keep {bar} around and deprecate it instead? Then we can remove it in a future release.

Reply:
If not possible then document the breaking change in NEWS.md. I started it off with a couple of examples from the other blocks.